### ISSUE TITLE
Wrong record is deleted

### ISSUE THREAD

**Issue by TacoScheltema** [ngoduykhanh/PowerDNS-Admin#53]
Hi,

I've setup a domain with the following records (domain names and ip addresses have been changed here for the example of course)
```
@       A       Active       3600       192.168.0.1
@       MX      Active       3600       10 mx1.example.com 
@       MX      Active       3600       40 mx2.example.com 
@       NS      Active       3600       ns1.example.com 
@       NS      Active       3600       ns2.example.com
@       SPF     Active       3600       "v=spf1 a mx ptr mx:mail.example.com ~all" 
@       TXT     Active       3600       "v=spf1 a mx ptr mx:mail.example.com ~all" 
www	    CNAME	Active       3600       web.example.com
```
When I then try to delete a record it's always deleting the topmost record instead of the selected record.

I am using Powerdns 3.4.7
```
dpkg -l | grep pdns
ii  pdns-backend-mysql                   3.4.7-1~bpo8+1              amd64        generic MySQL backend for PowerDNS
ii  pdns-server                          3.4.7-1~bpo8+1              amd64        extremely powerful and versatile nameserver
```

I did a `git pull origin` earlier today.
Is this something in my setup or could this be a bug?

**Comment by ngoduykhanh** [comment]
Fixed in commit 50973a295c9dbbcf1be18f004366f7d5662668a7. Thanks for reporting.

**Comment by TacoScheltema** [comment]
Yes, it's working now. Thanks for the quick fix 👍
